lib/pageflow/configuration/permissions.rb

```diff
@@ -0,0 +1,30 @@
+module Pageflow
+  class Configuration
+    # Permissions related options to be defined in the pageflow
+    # initializer of the main app.
+    class Permissions
+      def initialize(options = {})
+        @show_entry_features_to_non_admin =
+          options.fetch(:show_entry_features_to_non_admin, true)
+        @show_admin_boolean_to_non_admin =
+          options.fetch(:show_admin_boolean_to_non_admin, true)
+        @show_theming_dropdown_to_non_admin =
+          options.fetch(:show_theming_dropdown_to_non_admin, true)
+      end
+
+      # Allow non-admin users to see entry features. Defaults to true.
+      # @since edge
+      attr_accessor :show_entry_features_to_non_admin
```

Comment:
In case I hear no objection, I shall go with `non_admin_may_update_features` here. Reasoning: - name is more likely to remain valid even if we adjust roles and permissions so that `account_managers` prefix became a misnomer
- `may` is consistent with this being a permission. `can` implies that we assume our authorization layer is impervious to attacks -- which we do not
- `may` may unlock our thinking from CanCan-like abilities